### PR TITLE
fix: box/fixed storybook accessibility violation 

### PR DESCRIPTION
**Pull request by AnshumanPadiya**
Removes the storybook accessibility violation for Box/Fixed 

#### Where should the reviewer start?
/components/Box/Fixed.js

#### What testing has been done on this PR?
`yarn test`

#### Do Jest tests follow these best practices?

- [ ] `screen` is used for querying.
- [ ] The correct query is used. (Refer to [this list of queries](https://testing-library.com/docs/queries/about/#priority))
- [ ] `userEvent` is used in place of `fireEvent`.
- [ ] `asFragment()` is used for snapshot testing.

#### What are the relevant issues?
#6124 

#### Screenshots (if appropriate)
box-fixed-accessibility-violation

#### Do the grommet docs need to be updated?
up to the maintainers to decide

### CODE REVIEW

**Comment by AnshumanPadiya** [comment]
> 
Sure thing! I wonder how it got changed, I am really sorry though I didn't notice that change.

**Comment by jcfilben** [comment]
> Sure thing! I wonder how it got changed, I am really sorry though I didn't notice that change.

No worries!

**Review by jcfilben — APPROVED** [review]
Looks good!

**Review by ericsoderberghp — APPROVED** [review]

**Comment by AnshumanPadiya** [comment]
@jcfilben Hey, I noticed the test failure but still cannot understand the reason behind it, since there was no change in the DateInput component. I could be wrong about it since I cannot figure this out. I would be glad if you could help me with this.

**Comment by jcfilben** [comment]
> @jcfilben Hey, I noticed the test failure but still cannot understand the reason behind it, since there was no change in the DateInput component. I could be wrong about it since I cannot figure this out. I would be glad if you could help me with this.

We ran into an issue with one of our DateInput tests when we moved into a new month. The issue has been fixed in this pr: https://github.com/grommet/grommet/pull/6471 and if we update this branch the test should no longer fail. I will go ahead and update the branch and then I think this PR is ready to go!

**Comment by AnshumanPadiya** [comment]
> > @jcfilben Hey, I noticed the test failure but still cannot understand the reason behind it, since there was no change in the DateInput component. I could be wrong about it since I cannot figure this out. I would be glad if you could help me with this.
> 
> We ran into an issue with one of our DateInput tests when we moved into a new month. The issue has been fixed in this pr: #6471 and if we update this branch the test should no longer fail. I will go ahead and update the branch and then I think this PR is ready to go!

That's great! Thanks for helping.

File: src/js/components/Box/stories/Fixed.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export const FixedSizesBox = () => (
       justify="center"
       background="brand"
       overflow={{ horizontal: 'hidden', vertical: 'scroll' }}
+      tabIndex={0}
```
[the remark this answers is not in the document]
Comment (jcfilben) [reply src/js/components/Box/stories/Fixed.js:17]: See https://github.com/grommet/grommet/issues/6484

**Pull request merged**